skt/reporter.py

```diff
@@ -519,16 +475,11 @@ class StdioReporter(Reporter):
     TYPE = 'stdio'
 
     def report(self, printer=sys.stdout):
-        if self.multireport:
-            # We need to run the reporting function first to get the aggregated
-            # data to build subject from
-            report = self._get_multireport()
-            printer.write("{}\n".format(self._get_multisubject()))
-            printer.write(report)
-        else:
-            self._update_mergedata()
-            printer.write("Subject: {}\n".format(self._getsubject()))
-            printer.write(self._getreport())
+        # We need to run the reporting function first to get the aggregated
+        # data to build subject from
+        report = self._get_multireport()
+        printer.write("Subject: {}\n".format(self._get_multisubject()))
```

Comment:
Don't know if it was on purpose but it isn't exactly the same code than before, you've added `Subject:  `

Reply:
The stdio reporter didn't have `Subject:` for multireports and that broke the tests.

Reply:
Hehe :man_facepalming: ok